### ISSUE TITLE
Retrospective: make the sprint board so that everything moves left to right.

### ISSUE THREAD

**Issue by mreekie** [IQSS/dataverse#9202]
Retrospective outcome: make the sprint board so that everything moves left to right.

The reason is that it's confusing seeing PRs that are new and those made by the team in the "this sprint column".

Def of done:
- [ ] Add a "ready for review" column
- [ ] Define a process for moving PRs, that are generated by the team via an issue during the sprint, into the "ready for review" column
- [ ] Define a process for moving PRs, that are joining the sprint from the "ordered backlog" at sprint kickoff, into the "This Sprint" column.
- [ ] Answer the question - will automation work in this scenario?

**Comment by mreekie** [comment]
Notes:
Automation may be difficult in this situation because the system will interpret PRs that are added at the sprint kickoff and PRs that are added as a result of work done in the sprint as "New" when it first sees them.


If we use automation:
- Maybe we could make the automation always move PRs to the "ready for review" column, and manually move those PRs back to the "This Sprint Column" before the kickoff.

- Maybe we could just let the automation always move PRS to the "ready for review" column and be happy with that.  
The reasoning here is:
 - Any PR on the board is already sized, either as part of it's issue or on it's own as a community PR. 
   - In other words, by definition we intended it to be there at the beginning of the sprint.
 - As along as items move left to Right, where they start in the sprint won't matter. i.e. as a dev I can pick up something in the "ready for review" column. As a dev I can pick up something in the "This sprint" column.

**Comment by mreekie** [comment]
In looking at the sizing of this, I think regardless of the approach this is less than a week calendar time. Maybe less than a day.

**Comment by pdurbin** [comment]
Spoke with @mreekie 

I deleted the "Added To Board By Mistake" column. The automation especially was very confusing. We are always adding PRs to this board and this board was making it look like we are making mistakes when we are not.

I created at "Ready for Review" column. Now work can go from "In Progress" to "Ready for Review" to "Review". Also, I set up automation so that new PRs will go to the new "Ready for Review" column.

Closing.
